### ISSUE TITLE
[bitwarden-cli 1.17.0] Cannot find module 'tldjs'

### ISSUE THREAD

**Issue by chenrui333** [bitwarden/cli#342]
## Describe the Bug

During bitwarden-cli 1.17.0 release build, I run into some issue with the regression test:

```
==> /opt/homebrew/Cellar/bitwarden-cli/1.17.0/bin/bw generate --length 10
node:internal/modules/cjs/loader:930
  throw err;
  ^

Error: Cannot find module 'tldjs'
```

relates to:
- https://github.com/Homebrew/homebrew-core/pull/80324

**Comment by latipun7** [comment]
I have an issue too. Run `bw --help` error too
```bash
❯ bw --help
internal/modules/cjs/loader.js:905
  throw err;
  ^

Error: Cannot find module 'tldjs'
Require stack:
- /home/latipun7/.fnm/node-versions/v14.17.1/installation/lib/node_modules/@bitwarden/cli/build/bw.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at Function.Module._load (internal/modules/cjs/loader.js:746:27)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:92:18)
    at Object.<anonymous> (/home/latipun7/.fnm/node-versions/v14.17.1/installation/lib/node_modules/@bitwarden/cli/build/bw.js:140:18)
    at __webpack_require__ (/home/latipun7/.fnm/node-versions/v14.17.1/installation/lib/node_modules/@bitwarden/cli/build/bw.js:21:30)
    at Module.<anonymous> (/home/latipun7/.fnm/node-versions/v14.17.1/installation/lib/node_modules/@bitwarden/cli/build/bw.js:907:23)
    at __webpack_require__ (/home/latipun7/.fnm/node-versions/v14.17.1/installation/lib/node_modules/@bitwarden/cli/build/bw.js:21:30)
    at /home/latipun7/.fnm/node-versions/v14.17.1/installation/lib/node_modules/@bitwarden/cli/build/bw.js:85:18
    at Object.<anonymous> (/home/latipun7/.fnm/node-versions/v14.17.1/installation/lib/node_modules/@bitwarden/cli/build/bw.js:88:10) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/latipun7/.fnm/node-versions/v14.17.1/installation/lib/node_modules/@bitwarden/cli/build/bw.js'
  ]
}
```
OS: Linux WSL (Linux DESKTOP-GVN6OUJ 4.19.104-microsoft-standard #1 SMP Wed Feb 19 06:37:35 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux)

**Comment by eliykat** [comment]
Thanks, I'm also experiencing this behaviour with the npm package. We'll prioritise it for a fix.

**Comment by chenrui333** [comment]
@Hinton thanks for the quick fix, are we gonna make a quick patch release?

**Comment by eliykat** [comment]
@chenrui333 yes! A patch release is in the works.

**Comment by chenrui333** [comment]
Sounds good, btw, since the patch works for me, I went ahead applying it for the build and released 1.17.0 in the homebrew side.
